app/views/api/integrations/apicast/configuration_driven/_apicast.html.slim

```diff
@@ -1,53 +1,56 @@
-section.Configuration.Configuration--is-promoted class="Configuration--#{@show_presenter.test_state_modifier}"
+section.Configuration class=('Configuration--is-promoted' unless @service.deployment_option =~ /^service_mesh/) class="Configuration--#{@show_presenter.test_state_modifier}"
 
   = link_to 'edit APIcast configuration', edit_admin_service_integration_path(@service), class: 'SettingsBox-toggle'
 
-  article.Configuration-summary data-state="open" class="Environment--#{@show_presenter.test_state_modifier}"
+  article.Configuration-summary data-state="open" class=("Environment--#{@show_presenter.test_state_modifier}" unless @service.deployment_option =~ /^service_mesh/)
     h3.Configuration-title APIcast Configuration
 
     dl.u-dl
-      dt.u-dl-term Private Base URL
-      dd.u-dl-definition = @proxy.api_backend
+      - unless @service.deployment_option =~ /^service_mesh/
+        dt.u-dl-term Private Base URL
+        dd.u-dl-definition = @proxy.api_backend
       dt.u-dl-term Mapping rules
       dd.u-dl-definition
         - if (last_rule = @proxy.proxy_rules.last)
           code
             ' #{last_rule.pattern} => #{last_rule.metric.name}
         - if (rules_size = @proxy.proxy_rules.size) > 1
           | and #{rules_size - 1} more.
-      dt.u-dl-term Credential Location
-      dd.u-dl-definition = @proxy.credentials_location
-      dt.u-dl-term Secret Token
-      dd.u-dl-definition = @proxy.secret_token
+      - unless @service.deployment_option =~ /^service_mesh/
+        dt.u-dl-term Credential Location
+        dd.u-dl-definition = @proxy.credentials_location
+        dt.u-dl-term Secret Token
+        dd.u-dl-definition = @proxy.secret_token
 
-section style="margin-top: 48px;"
-  h2 Environments
+- unless @service.deployment_option =~ /^service_mesh/
```

Comment:
I see this conditional `@service.deployment_option =~ /^service_mesh/` repeated 5 times in 25 lines of code. Maybe a helper?